supabase_client.py

```
@@ -0,0 +1,92 @@
+
+import logging
+import os
+from typing import Optional, Tuple
+from supabase import create_client, Client
+import config
+
+# Initialize Supabase client
+try:
+    supabase: Client = create_client(config.SUPABASE_URL, config.SUPABASE_ANON_KEY)
+    logging.info("Supabase client initialized successfully.")
+except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)